### PR TITLE
[AU-895] Add new documentation

### PR DESCRIPTION
**Pull request by Mozart2234**
### Description
Give a brief description here of what you're trying to accomplish and why it's needed.

Document the ability to specify `assigned_stall` to the list of associations to render/include in the payload for `GET /api/v3/invites/:id`.

### Changelog
- Adds the **_invites_parking_lot_** with his attributes in the invite detail

## Notes (can delete this section after doing):
- [ ] Start at least one of your commits with your wrike ticket in the following format, `[#1234] Your concise and explicit commit message`
- [ ] Use the `Squash and Merge` button to combine your commits in to one final commit (with the wrike ticket) to go in to master

### CODE REVIEW

**Review by nickrobertson — APPROVED** [review]

**Review by nucleom42 — DISMISSED** [review]

**Review by diegotoral — APPROVED** [review]

**Review by nucleom42 — APPROVED** [review]

**Review by pluus — DISMISSED** [review]

**Review by diegotoral — DISMISSED** [review]

**Review by nucleom42 — DISMISSED** [review]

**Review by amaurymartin — DISMISSED** [review]
looks good! I'm assuming that when you merge this PR, it will update the content on https://developers.signinenterprise.com/, so keep eye on it and make sure everything looks good after merging 🙂

**Review by diegotoral — APPROVED** [review]

**Review by ErebusBat — APPROVED** [review]

File: models/InviteCreateParams.v1.yaml [hunk 1]
```diff
@@ -84,8 +88,15 @@ properties:
     type: string
   group_visit_id:
     type: integer
-  license_plate:
-    type: string
+  parking_stall:
+    type: object
```
Comment (diegotoral) [inline models/InviteCreateParams.v1.yaml:92]: Can it be `null`?

File: models/InviteDetail.v1.yaml [hunk 2]
```diff
@@ -111,6 +111,13 @@ x-examples:
         watchlist_level: some text
         created_at: '2018-02-10T09:30Z'
         updated_at: '2018-02-10T09:30Z'
+      invites_parking_lot:
```
Comment (diegotoral) [inline models/InviteDetail.v1.yaml:114]: I thought the property was `assigned_stall`. Did it change? 🤔

**Review by diegotoral — APPROVED** [review]

File: models/InviteDetail.v1.yaml (hunk 2 above)
Comment (Mozart2234) [reply models/InviteDetail.v1.yaml:114]: this section is for the response.

**Pull request merged**